RELEASING.md

```diff
@@ -1,12 +1,15 @@
 Releasing
 =========
 
+Note that these steps require pushing directly to the master branch on GitHub.  Be sure that other developers are not pushing changes concurrently.
+
  1. Change the version in `gradle.properties` to a non-SNAPSHOT version.
- 2. Update all README files with the new version.
+ 2. Update all README files (except `README-developers.md`) with the new release version.
```

Comment:
I suggest combining points 1 and 2 since they are both setting a new version in some files.